omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -637,7 +637,7 @@ def generate_rabbit_actions_password
                         value: PrivateChef["external_rabbitmq"]["actions_password"],
                         frozen: true, force: true)
       else
-        credentials.add("rabbitmq", "actions_password", length: 100)
+        add_secret({group: "rabbitmq", name: "actions_password", length: 100}, true)
```

Comment:
i think you can use `add_secret(group: "rabbitmq", name: "actions_password", length: 100)` here